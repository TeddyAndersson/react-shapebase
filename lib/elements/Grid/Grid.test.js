import React from 'react';
import Grid from './Grid'
import renderer from 'react-test-renderer';

test('should render Grid', () => {
  const component = renderer.create(
    <Grid>
      <Grid.Item>
        Item one
      </Grid.Item>
      <Grid.Item>
        Item two
      </Grid.Item>
      <Grid.Item>
        Item three
      </Grid.Item>
      <Grid.Item>
        Item four
      </Grid.Item>
    </Grid>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})
