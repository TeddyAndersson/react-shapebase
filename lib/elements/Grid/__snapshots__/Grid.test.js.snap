// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Grid 1`] = `
<div
  className="sc-bwzfXH gYzJkf"
>
  <div
    className="sc-bdVaJa QaxH"
  >
    Item one
  </div>
  <div
    className="sc-bdVaJa QaxH"
  >
    Item two
  </div>
  <div
    className="sc-bdVaJa QaxH"
  >
    Item three
  </div>
  <div
    className="sc-bdVaJa QaxH"
  >
    Item four
  </div>
</div>
`;
